### PR TITLE
More XR selector fixes

### PR DESCRIPTION
**Pull request by Toseben**
Able to switch between characters without crash
Removed extra empty rows from selector
Fixed some issues with character and bones scale caused by going from THREE ID to Shot Generator ID

### CODE REVIEW

File: src/js/express-xr/src/SceneManagerXR.js [hunk 1]
```diff
@@ -309,13 +310,21 @@ const SceneContent = ({
...
+        // hacky way of refreshing slider values
+        setSelectedObject(0)
+        setSelectedObject(object.userData.id)
       }
 
       if (intersection.object.name.includes('selector-character')) {
         const model = intersection.object.name.split('_')[1]
         const object = worldScaleGroupRef.current.children.find(child => child.userData.id === selectedObject)
-        updateObject(object.userData.id, { model })
+        updateObject(object.userData.id, { model, ['height']: initialState.models[model].height })
```
Comment (audionerd) [inline src/js/express-xr/src/SceneManagerXR.js]:
```suggestion
        updateObject(object.userData.id, { model, height: initialState.models[model].height })
```

File: src/js/express-xr/src/components/SGCharacter.js [hunk 2]
```diff
@@ -494,7 +494,7 @@ const SGCharacter = React.memo(({ id, type, worldScale, isSelected, updateObject
           >
             <primitive userData={{ id, type }} object={skinnedMesh} />
             <primitive object={armatures[0]} />
-            {props.children}
+            {bonesHelper && <group>{props.children}</group>}
```
Comment (audionerd) [inline src/js/express-xr/src/components/SGCharacter.js]: Do the `children` need to be in a `Group`?

Comment (Toseben) [reply src/js/express-xr/src/components/SGCharacter.js]: Tried without  `Group` and doesn't seem to change anything!

Comment (audionerd) [reply src/js/express-xr/src/components/SGCharacter.js]:
```suggestion
            {bonesHelper && props.children}
```

**Pull request merged**